docs/whitelabelling.md

```diff
@@ -0,0 +1,105 @@
+# Whitelabelling
+
+The aim of this document is to provide a (preferably) complete overview about the process of creating a new whitelabelled app.
+
+## Contents
+* [Necessary information from city](#from-city)
+* [Necessary information from CMS](#from-cms)
+* [AppTeam TODOs](#appteam-todos)
+
+## Necessary Information
+
+A list of necessary information to build and deliver a whitelabelled app.
+
+### From City
+
+Information the city needs to provide for their own whitelabelled app.
+Some points may overlap and are therefore duplicated.
+
+#### For build config
+
+Information necessary for the creation of a new build config.
+
+* App name
+* Application id / bundle identifier
+* App icon
+* Header icon
+* Theme color
+* Activated features
+* Splash screen / launch screen (+ background for webapp)
+* Imprint text
+* About app link
+* Privacy policy link
+* Languages of the app
+* Different translations for locales (especially metadata)
+* Deactivated features: pois, news stream, push notifications, offers, ...
+* Fonts (Content, UI only at own risk)
+
+#### For Stores/WebApp
+
+Information necessary for the setup of the store and webapp presences.
+
+* WebApp domain
```

Comment:
Where/How to deploy the webapp. Do we host it? Probably yes...

Reply:
Question added